scripts/screen_record.sh

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env bash
+# Requires wf-recorder: https://github.com/ammen99/wf-recorder
 
 outputDir="$HOME/Videos/Screencasts"
+defaultSink=$(pactl info | grep "Default Sink" | awk -F ': ' '{print $NF}')
+WF_RECORDER_OPTS="--audio $defaultSink --codec av1_vaapi"
```

Comment:
```suggestion
defaultSink=$(pactl get-default-sink)
WF_RECORDER_OPTS="--audio=$defaultSink.monitor -c libx264rgb"
```
- Changed the default audio input to be the desktop audio of the default sink.
- Changed the video codec to software instead of hardware accelerated for now as it works more out of the box. We can make this more configurable in a separate PR